microsoft-edge/webview2/how-to/prerelease-testing.md

```diff
@@ -0,0 +1,258 @@
+---
+title: Prerelease testing using preview channels
+description: How to test your WebView2 app by using the preview channels of Microsoft Edge (Edge Canary, Edge Dev, or Edge Beta) to find issues that will affect your particular WebView2 app, before WebView2 changes migrate to the Evergreen WebView2 Runtime.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.subservice: webview
+ms.date: 02/15/2024
+---
+# Prerelease testing using preview channels
+
+_Prerelease testing_ means testing your WebView2 app by using the preview channels of Microsoft Edge to find issues that will affect your particular WebView2 app, before WebView2 changes migrate to Evergreen WebView2 Runtime.  Prerelease testing helps your app catch bugs in prerelease channels of WebView2, before the WebView2 changes ship in a stable release and degrade the app experience for end users.  In prerelease testing, you validate your app against early builds of WebView2, to ensure that your app will work with the upcoming changes.
+
+To validate your app against early builds of WebView2, do automated testing with the preview channels of Microsoft Edge.  A preview channel contains a WebView2 preview runtime.
+
+The preview (or "insider") channels of Microsoft Edge consist of:
+* Edge Canary - releases daily.
+* Edge Dev - releases weekly.
+* Edge Beta - releases every 4 weeks.
+
+See also: 
+* [Overview of the Microsoft Edge channels](/deployedge/microsoft-edge-channels)
+* [Become a Microsoft Edge Insider](https://www.microsoft.com/edge/download/insider) - download Edge Canary, Edge Dev, or Edge Beta
+* [Self-hosting by deploying preview channels](./self-hosting.md)
+
+
+<!-- ====================================================================== -->
+## Importance of prerelease testing
+
+Evergreen WebView2 is based on the evergreen Chromium platform, which receives monthly major updates.  To validate your own, particular app against incoming browser updates, test your WebView2 app against the prerelease channels of Microsoft Edge (Edge Canary, Edge Dev, or Edge Beta).  Regressions in WebView2 apps tend to be application-specific, rather than affecting all WebView2 apps.
```

Comment:
> test your WebView2 app against the prerelease channels of Microsoft Edge (Edge Canary, Edge Dev, or Edge Beta).

Test your WebView2 app against the prerelease runtime shipped with the prerelease channels of Microsoft Edge  (Edge Canary, Edge Dev, or Edge Beta).

I prefer the above phrasing, because I'm afraid people might be confused as to the fact that Edge is not a runtime.

Reply:
```
To catch app-specific regressions before WebView2 changes ship in Edge Stable, test your WebView2 app against the prerelease runtime that's shipped with the prerelease channels of Microsoft Edge (Edge Canary, Edge Dev, or Edge Beta).
```